groovy/postInit/chemistry/elements/MagnesiumChain.groovy

```diff
@@ -40,6 +49,20 @@ EBF.recipeBuilder()
         .blastFurnaceTemp(1370)
         .buildAndRegister()
 
+for (carbon_source in carbon_sources) {
```

Comment:
we use Globals.combustibles or Globals.highPurityCombustibles if just carbon/coke is allowed.